ci/pipeline.yaml

```diff
@@ -0,0 +1,72 @@
+resource_types:
+  - name: cf-cli
+    type: docker-image
+    source:
+      repository: nulldriver/cf-cli-resource
+
+resources:
+  - name: di-authentication-api
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/alphagov/di-authentication-api.git
+      ignore_paths:
+        - ci/pipeline.yaml
+      branch: main
+
+  - name: pipeline-src
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/alphagov/di-authentication-api.git
+      paths:
+        - ci/pipeline.yaml
+      branch: main
+
+  - name: di-authentication-api-upload
+    type: cf-cli
+    icon: cloud-upload
+    source:
+      api: https://api.london.cloud.service.gov.uk
+      username: ((cf-username))
```

Comment:
We probably need to give the user access to the `build` space in PaaS

Reply:
Good shout Alex. Let me check that!

Reply:
I've given everyone access now. Cheers for that!